### PR TITLE
chore: update "01 Fetchers" problems from README.md 

### PR DESCRIPTION
**Pull request by dev-xo**
Hello @kentcdodds!

Here's the discussed PR from Discord. Let me know if I should add some brief description about the video on top of the `VideoEmbed`, or if it's good enough like this.

Thanks!

### CODE REVIEW

File: exercises/01.cookies/01.problem.fetcher/README.mdx [hunk 1]
```diff
@@ -63,3 +63,8 @@ to add a `console.log` in the action and have the request data there when you're
 done.
 
 - [📜 `useFetcher`](https://remix.run/docs/en/main/hooks/use-fetcher)
+
+<VideoEmbed
+	title="Understanding the differences between Form vs. Fetcher"
+	url="https://youtu.be/1bsNKdMiph0"
+/>
```
Comment (kentcdodds) [inline exercises/01.cookies/01.problem.fetcher/README.mdx]: Let's add a little more to this and also put it above the footer links :)

```suggestion
`useFetcher()` and the `<Form />` component + `useNavigation` hook appear to have
some overlapping use cases. If your confused about when to use one or the other,
give this quick video by [Daniel Kanem](https://twitter.com/DanielKanem) a watch:

<VideoEmbed
	title="Understanding the differences between Form vs. Fetcher"
	url="https://youtu.be/1bsNKdMiph0"
/>

- [📜 `useFetcher`](https://remix.run/docs/en/main/hooks/use-fetcher)
```

**Review by kentcdodds — APPROVED** [review]
Let's expand on this a bit :)

**Review by kentcdodds — APPROVED** [review]
Thanks!

**Comment by kentcdodds** [comment]
Whoops, this should have been the embed URL for youtube 😅

**Comment by kentcdodds** [comment]
I'll fix it. No worries :)

**Comment by kentcdodds** [comment]
Done: https://github.com/epicweb-dev/web-auth/commit/cc743f42817bc7f76a478cd257e160282ded224f

**Comment by dev-xo** [comment]
Thanks, Kent!

Sorry I was unable to be awake for this; otherwise, I would have helped with it. Let me know if you need some fancy animations or something related anytime!

**Pull request merged**